tonic/src/request.rs

```diff
@@ -7,6 +8,18 @@ pub struct Request<T> {
     message: T,
 }
 
+/// A trait that is implemented for all streaming RPC request types.
```

Comment:
This doc comment should probably also describe what the trait is _used_ for, not just what types implement it. Something like
> Types implementing this trait may be used as an argument to generated client-streaming RPC methods